workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/fragments/group/marketplace-base-fragments/fragments/marketplace-account-menu/fragment.json

```diff
@@ -0,0 +1,9 @@
+{
+	"configurationPath": "configuration.json",
+	"cssPath": "index.css",
+	"htmlPath": "index.html",
+	"icon": "code",
+	"jsPath": "index.js",
+	"name": "Marketplace account menu",
```

Comment:
Let's not add additional context, you can also rename the folder name to `account-menu` instead of `marketplace-account-menu`

```suggestion
	"name": "Account Menu",
```

Reply:
Use already existing `account-menu`